router/router.go

```diff
@@ -32,6 +32,8 @@ const (
 
 	ServiceNowTableDefault = "incident"
 	AnonymizeReplacement   = "<hidden>"
+
+	customTriggerTypeField = "custom_trigger_type"
```

Comment:
Instead of `customTriggerTypeField = "custom_trigger_type"` it should be
`resourceTypeKey  = "resourceTypeKey"`

Reply:
Done